experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -490,6 +490,7 @@ def resolve_status_update_exempt_fields(self, info):
 
 class NimbusExperimentType(DjangoObjectType):
     application = graphene.NonNull(NimbusExperimentApplicationEnum)
+    audience_url = graphene.String()
```

Comment:
This should be non-null, then we dont need the `audienceUrl!` in TS.
```suggestion
    audience_url = graphene.NonNull(graphene.String)
```